client/my-sites/plugins-wpcom/business-plugins-panel.jsx

```diff
@@ -18,7 +19,9 @@ const defaultPlugins = [
 
 export const BusinessPluginsPanel = React.createClass( {
 	render() {
-		const { plugins: givenPlugins = [] } = this.props;
+		const { plugins: givenPlugins = [], siteSlug } = this.props;
+		defaultPlugins[0].featureDescriptionLink = `/plans/features/google-analytics/${ siteSlug }`;
```

Comment:
I wonder if there isn't a better way to handle this. Maybe a template placeholder to replace, maybe a function to build it?

```js
const siteLink = ( siteSlug, url ) => path( url ).appendPath( siteSlug );
```

or something like above…